package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "glorious-pins",
+  "version": "0.0.1",
```

Comment:
In every new Pull Request should we increase the version number? What do you say?